R/moma_arguments.R

```diff
@@ -1,80 +1,83 @@
 #' Sparsity-inducing penalty in \code{MoMA}
+#'
 #' In the package \code{MoMA}, we support the following sparsity-inducing
 #' penalty functions.
 #' \itemize{
-#'   \item{\code{\link{lasso}}} TODO
-#'   \item{\code{\link{mcp}}} TODO
-#'   \item{\code{\link{scad}}} TODO
-#'   \item{\code{\link{slope}}} TODO
-#'   \item{\code{\link{grplasso}}} TODO
-#'   \item{\code{\link{fusedlasso}}} TODO
-#'   \item{\code{\link{l1tf}}} TODO
-#'   \item{\code{\link{spfusedlasso}}} TODO
-#'   \item{\code{\link{cluster}}} TODO
+#'   \item{\code{\link{moma_lasso}}}: sparsity
+#'   \item{\code{\link{moma_mcp}}}: adaptive sparsity
+#'   \item{\code{\link{moma_scad}}}: adaptive sparsity
+#'   \item{\code{\link{moma_slope}}}: adaptive sparsity
+#'   \item{\code{\link{moma_grplasso}}}: group-wise sparsity
+#'   \item{\code{\link{moma_fusedlasso}}}: piece-wise constant
+#'   \item{\code{\link{moma_spfusedlasso}}}: sparsity and piece-wise constant
+#'   \item{\code{\link{moma_l1tf}}}: piece-wise quadratic
```

Comment:
"Piecewise polynomial (default quadratic)" might be more accuracy.

Reply:
Fixed.